src/main/java/de/tum/in/www1/artemis/web/rest/publicc/BuildPlanResource.java

```diff
@@ -1,22 +1,20 @@
-package de.tum.in.www1.artemis.web.rest;
-
-import static de.tum.in.www1.artemis.web.rest.ProgrammingExerciseResourceEndpoints.BUILD_PLAN;
+package de.tum.in.www1.artemis.web.rest.publicc;
```

Comment:
We plan to extend this resource somewhen in the near future with more endpoints that should not be public (e.g. allow editors/instructors to edit the build plan in the Artemis UI).
```suggestion
package de.tum.in.www1.artemis.web.rest;
```

Reply:
As you might see, I moved all public endpoints to their respective resources. Once you create other endpoints, feel free to create a new resource outside the `publicc` package. You can then also rename this class to `PublicBuildPlanResource` following the other names just to have a better distinction.

Hence, I will resolve your other conversations regarding this too.
As this whole thing is about separating these different types, it makes, in my opinion, also sense to separate them within the code base.